src/messages/MMgrBeacon.h

```diff
@@ -169,7 +179,23 @@ class MMgrBeacon final : public PaxosServiceMessage {
       decode(mgr_features, p);
     }
     if (header.version >= 10) {
-      decode(clients, p);
+      std::vector<entity_addrvec_t> clients_addrs;
+      decode(clients_addrs, p);
+      clients.clear();
+      if (header.version >= 11) {
+	std::vector<std::string> clients_names;
+	decode(clients_names, p);
+	assert(clients_names.size() == clients_addrs.size());
```

Comment:
Is it OK to assert?

Reply:
No, I think `ceph::buffer::malformed_input` with a short description string should be thrown instead.

Reply:
Thanks! Done.  Let me know if the description string looks okay?

Reply:
> Let me know if the description string looks okay?

It does!